lib/winston/transports/http.js

```diff
@@ -49,17 +51,29 @@ module.exports = class Http extends TransportStream {
    * @returns {undefined}
    */
   log(info, callback) {
-    this._request(info, (err, res) => {
-      if (res && res.statusCode !== 200) {
-        err = new Error(`Invalid HTTP Status Code: ${res.statusCode}`);
-      }
-
-      if (err) {
-        this.emit('warn', err);
-      } else {
-        this.emit('logged', info);
-      }
-    });
+    const index = this.callIndex++;
```

Comment:
Should this be reset at Max_Safe_Integer?   It might be an edge case but it depends on the amount of logging being done.